docs/operator-manual/installation.md

```diff
@@ -46,15 +46,15 @@ High Availability installation is recommended for production use. Bundle include
 * [ha/namespace-install.yaml](https://github.com/argoproj/argo-cd/blob/master/manifests/ha/namespace-install.yaml) - the same as namespace-install.yaml but
   with multiple replicas for supported components.
 
-## Single-Tenant (a.k.a. "Headless")
+## Single-Tenant (a.k.a. "Core")
 
-The single-tenant installation is most suitable for cluster administrators who indepently use Argo CD and don't need multi-tennancy features. This installation
+The single-tenant/core installation is most suitable for cluster administrators who indepently use Argo CD and don't need multi-tennancy features. This installation
```

Comment:
I think I would rename the title of the section from "single-tenant (AKA core)" to just "Core" installation. The fact that this is useful for single-tenancy can be mentioned in the description of when you would want to use this mode of installation.

typo: tenancy

Reply:
agree - fixed.